instrument-functions.c

```diff
@@ -158,14 +158,12 @@ static void print_debug(void *this_fn, void *call_site, action_type action)
 
 	if (instrument_global) {
 		symbol_info syminfo;
-		int found;
-
-		i = 0;
-		found = 0;
-		while (i < symcount && !found) {
+		int found = 0;
+		for (i = 0; i < symcount; i++) {
```

Comment:
Any reason to change this file?
This change add bug.

Reply:
It does not if we break immediately after?